test/applications/test_clique.py

```diff
@@ -0,0 +1,73 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2018, 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+""" Test Clique class"""
+
+from test.optimization_test_case import QiskitOptimizationTestCase
+import networkx as nx
+
+from qiskit_optimization import QuadraticProgram
+from qiskit_optimization.algorithms import (OptimizationResult,
+                                            OptimizationResultStatus)
+from qiskit_optimization.applications.clique import Clique
+from qiskit_optimization.problems import (Constraint, QuadraticObjective, VarType)
+
+
+class TestClique(QiskitOptimizationTestCase):
```

Comment:
Since `size` is a public method (getter/setter) there should be unit tests that validate it works as expected etc.